chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/impl/FilesDAOImpl.java

```diff
@@ -0,0 +1,122 @@
+package gov.healthit.chpl.dao.impl;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.Query;
+
+import org.springframework.stereotype.Repository;
+
+import gov.healthit.chpl.auth.Util;
+import gov.healthit.chpl.dao.FilesDAO;
+import gov.healthit.chpl.dto.FileTypeDTO;
+import gov.healthit.chpl.dto.FilesDTO;
+import gov.healthit.chpl.entity.FileTypeEntity;
+import gov.healthit.chpl.entity.FilesEntity;
+import gov.healthit.chpl.exception.EntityCreationException;
+import gov.healthit.chpl.exception.EntityRetrievalException;
+
+@Repository(value = "filesDAO")
+public class FilesDAOImpl extends BaseDAOImpl implements FilesDAO {
+
+    @Override
+    public FilesEntity create(final FilesDTO dto) throws EntityCreationException, EntityRetrievalException {
+        FileTypeEntity type = new FileTypeEntity();
+        type.setId(dto.getFileType().getId());
+
+        FilesEntity insert = new FilesEntity();
+        insert.setFileData(dto.getFileData());
+        insert.setFileName(dto.getFileName());
+        insert.setContentType(dto.getContentType());
+        insert.setAssociatedDate(dto.getAssociatedDate());
+        insert.setCreationDate(new Date());
+        insert.setLastModifiedDate(new Date());
+        if (dto.getDeleted() != null) {
```

Comment:
We have this if/else in some other DAOs and I always thought it was odd. Why would you insert something with "deleted" as anything but false? Can it just setDeleted to false always here?